src/plugins/data-card-tool/components/case-attribute.tsx

```diff
@@ -100,24 +112,49 @@ export const CaseAttribute: React.FC<IProps> = observer(props => {
     setEditFacet("");
   };
 
-  const activateInput = (facet: EditFacet) => {
+  const activateInput = (facet: EditFacet, editing: boolean) => {
     setEditFacet(facet);
-    if (facet === "name"){
+    if (facet === "name" && !editing){
       setLabelCandidate(getLabel());
     }
-    if (facet === "value"){
+    if (facet === "value" && !editing){
       setValueCandidate(getValue());
     }
     setCurrEditAttrId(attrKey);
   };
 
+  function deleteAttribute(){
+    console.log("? deleteAttribute: ", attrKey);
```

Comment:
Remove console.log

Reply:
Thanks for noticing that issue on the attribute editing. Just pushed a fix.